src/System.Data.SqlClient/tests/ManualTests/SQL/RandomStressTest/Randomizer.cs

```diff
@@ -256,8 +256,8 @@ public int NextSeed()
             return Next();
         }
 
-        #region Random core implementation taken from .Net 4.0 source code- DO NOT CHANGE BEYOND THIS POINT
-        // The code below was copied from .Net 4.0 sources of Random class
+        #region DO NOT CHANGE BEYOND THIS POINT
+        // The code below was copied from https://github.com/dotnet/coreclr/blob/master/src/mscorlib/src/System/Random.cs
```

Comment:
I was wondering the same thing. Why can't the test just use the Random in the product. It's just a test after all so taking on a dependency (that it already has) shouldn't matter.